### PR TITLE
update limitations section in README

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [ ] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

Update limitations documentation after autochart logic changes.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -200,14 +200,14 @@ These are the current limitations in Gopherciser:
   * Variance waterfall chart
   * P&L pivot chart object
   * Trellis container extension
-  * Chart suggestions (that is, auto-charts) for box plot, distribution plot, and histogram objects (to get correct measurements for these types of objects, disable chart suggestions in each such object before starting your test)
+  * Chart suggestions (that is, auto-charts) are supported, however only if the objects where created in Qlik Sense June 2020 or later. For auto-chart objects created in earlier version, this needs to be updated manually in your app
```
Comment (SeBBBe) [inline docs/README.md:203]: * were created
* earlier version**s**
* **these** need to be (or maybe just "auto-chart objects created in earlier versions need to be updated manually")

Comment (SeBBBe) [reply docs/README.md:203]: Carl will probably re-write it :P

Comment (DnlLrssn) [reply docs/README.md:203]: that's likely, I will await his response.

Comment (carlcervin) [inline docs/README.md:203]: Suggestion: 
"Chart suggestions (that is, auto-charts) are supported, but only if the objects were created with Qlik Sense Enterprise June 2020 or later. Auto-chart objects created with earlier versions have to be manually updated in your app."

**Comment by SeBBBe** [comment]
Screenshot 2020-06-18 at 14 23 50

**Pull request merged**